src/polkadot/definitions.ts

```diff
@@ -1,5 +1,3 @@
-/* istanbul ignore file */
-
```

Comment:
This should be re-added by hand (or maybe by script), otherwise the coverage reports will be skewed